extension/encoding/awslogsencodingextension/internal/unmarshaler/elb-access-log/elb.go

```diff
@@ -0,0 +1,438 @@
+	BackendProcessingTime  float64 // Time taken for the registered instance to respond
+	ResponseProcessingTime float64 // Time taken to send the response to the client
+	ELBStatusCode          int64   // Status code from the load balancer
+	BackendStatusCode      int64   // Status code from the backend instance
+	ReceivedBytes          int64   // Size of the request in bytes received from the client
+	SentBytes              int64   // Size of the response in bytes sent to the client
+	RequestMethod          string  // HTTP method used in the request (e.g., GET, POST)
+	RequestURI             string  // Full URI requested by the client
+	ProtocolName           string  // Network protocol name (e.g., HTTP)
+	ProtocolVersion        string  // Network protocol version (e.g., 1.1)
+	UserAgent              string  // The User-Agent string identifying the client
+	SSLCipher              string  // The SSL cipher used, available for HTTPS listeners
+	SSLProtocol            string  // The SSL protocol negotiated, available for HTTPS listeners
+}
+
+// convertTextToClbAccessLogRecord converts a slice of strings into a ClbAccessLogRecord
+func convertTextToClbAccessLogRecord(fields []string) (ClbAccessLogRecord, error) {
+	var err error
+	fieldsCount := len(fields)
+	if fieldsCount < 15 {
+		return ClbAccessLogRecord{}, fmt.Errorf("clb access logs do not have enough fields. Expected 15, got %d", fieldsCount)
+	}
+
+	// Map fields to the struct
+	record := ClbAccessLogRecord{
+		Time:              fields[0],  // Timestamp
+		ELB:               fields[1],  // Load balancer name
+		BackendIpPort:     fields[3],  // Backend IP:Port
+		ELBStatusCode:     0,          // Placeholder for ELB status code
+		BackendStatusCode: 0,          // Placeholder for Backend status code
+		UserAgent:         fields[12], // User-Agent
+		SSLCipher:         fields[13], // SSL cipher
+		SSLProtocol:       fields[14], // SSL protocol
+	}
+
+	// Process the fields for numerical values (convenient to parse from string)
+	record.ClientIp = strings.Split(fields[2], ":")[0]
+	if record.ClientPort, err = safeConvertStrToInt(strings.Split(fields[2], ":")[1]); err != nil {
+		return record, fmt.Errorf("could not convert client port to integer: %w", err)
+	}
+	if record.RequestProcessingTime, err = safeConvertStrToFloat(fields[4]); err != nil {
+		return record, fmt.Errorf("could not convert request processing time to float: %w", err)
+	}
+
+	if record.BackendProcessingTime, err = safeConvertStrToFloat(fields[5]); err != nil {
+		return record, fmt.Errorf("could not convert backend processing time to float: %w", err)
+	}
+
+	if record.ResponseProcessingTime, err = safeConvertStrToFloat(fields[6]); err != nil {
+		return record, fmt.Errorf("could not convert response processing time to float: %w", err)
+	}
+
+	// ELB status code and backend status code can be - in case of TCP and SSL entry
+	if fields[7] != "-" {
```

Comment:
There is a constant for `-` so maybe use it here? And for all the others hardcoded `"-"`